designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There a some widely used ESLint related packages that aren't actively maintained
+anymore and we'd like to transfer them to a community owned GitHub organization
+(`@eslint-community`), so we can give them back to the community.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
+`@eslint-community` org would be a collection of important/widely used ESLint
+plugins/packages that aren't maintained anymore and are given back to the
+community.
+
+> Community repos for ESLint related projects
+
+The projects could be released under the `@eslint-community` scope in a later
+stage.
+
+The benefit of having the org in place would be that we could create a structure
+to streamline the packages more, i.e. tools used, supported Node/ESLint
```

Comment:
Should elaborate on `streamline` here.

Reply:
I mentioned things like using same tools, which Node/ESLint versions are supported, ...

What would you like to see more explained?

Reply:
@MichaelDeBoey I think eg. @aladdin-add is of a different perception on this. See: https://github.com/eslint/rfcs/pull/91#discussion_r885453500

Streamlining like this requires some level of collective maintainership at the org level rather than having each module maintained in isolation. Eg. @aladdin-add seems to be under the perception/belief that no such collective ownership will/should happen here.

Reply:
I'll make it more clear that we _could_ streamline things, but that it's not a goal on its own.

The main goal still remains to maintain abandoned ESLint related packages that are widely depended upon imo.